qutip/settings.py

```diff
@@ -122,21 +122,13 @@ class Settings:
     """
     Qutip's settings and options.
     """
-    def __new__(cls):
-        """Set Settings as a singleton."""
-        if not hasattr(cls, '_instance'):
-            cls._instance = super(Settings, cls).__new__(cls)
-        return cls._instance
-
     def __init__(self):
         self._mkl_lib = ""
         try:
             self.tmproot = os.path.join(os.path.expanduser("~"), '.qutip')
         except OSError:
             self._tmproot = "."
         self.core = None
```

Comment:
The `compile` attribute is sneakily added in coefficient.py. Can we add it here just as a note comment that it's will be added.
```suggestion
        self.core = None
        self.compile = None
```

Reply:
Good idea. I added `.compile` with a comment saying where the two are set.